_episodes/02-working-with-environments.md

```diff
@@ -359,23 +354,23 @@ $ conda install scikit-learn=1.0.2
 
 Environments created with `conda`, by default, live in the `envs/` folder of your `miniconda3` (or `anaconda3`)
 directory the absolute path to which will look something the following: `/home/$USERNAME/miniconda3/envs` (Linux),
-`/Users/$USERNAME/miniconda3/envs` (OSX) or `C:\Users\$USERNAME\Anaconda3` (Windows).
+`/Users/$USERNAME/miniconda3/envs` (OSX) or `C:\Users\$USERNAME\Anaconda3` (Windows). It's also been spotted in
+`C:\ProgramData\miniconda3\envs`.
 
 Running `ls` (Linux or OSX) / `dir` (Windows) on your anaconda `envs/` directory will list out the directories
 containing the existing Conda environments.
 
-> ## Location of Conda environments on Binder
->
-> If you are working through these lessons using a Binder instance, then the default location of
-> the Conda environments is slightly different.
+> ## Finding Conda
 >
+> If you're not sure where your Conda environments are stored, activate an environment, launch Python, then run
+> the following:
+> ```
+> import sys                   
+> sys.executable
```

Comment:
Would it be worth showing sample output here as well?

```
>>> import sys
>>> sys.executable
'/home/neil/miniconda3/bin/python'
```

Reply:
Yes - good idea.